tests/patients/test_patients.py

```diff
@@ -261,6 +261,13 @@ def test_query(app, workspace_generator):
     patients = pk.patients.query(workspace.id, "Test^1")
     assert len(patients) == 1
 
+    # Verify paging
+    for i in range(1000):
```

Comment:
This test now takes about 16s to finish due to the 1000 patients being created. @nathandieckhaus has a change to the API to accept a `page_size` which will allow this range to be a lot smaller. It might be best to wait for that PR to be merged.